### ISSUE TITLE
'ring' meter for SSD1322

### ISSUE THREAD

**Issue by momosh13** [olikraus/U8glib_Arduino#7]
Hi 
I find this article
http://www.instructables.com/id/Arduino-analogue-ring-meter-on-colour-TFT-display/
and i'm trying to convert to U8GLIB, for ssd1322, or maybe something like it, so i can 
have same type gauge display.
Please Help.
![maxresdefault](https://cloud.githubusercontent.com/assets/5689051/14583268/7e44f6b6-03d1-11e6-8662-cdfd94923806.jpg)

**Comment by olikraus** [comment]
SSD1322.... hmmm isn't this a monochrome display?

**Comment by momosh13** [comment]
yes, ssd1322 is monochrome, i'm trying to make gauge display for ssd1322, just like in the picture, only monochrome.
